extensions/amp-analytics/amp-analytics.md

```diff
@@ -59,6 +59,13 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
 </amp-analytics>
 ```
 
+#### <a name="attributes"></a>Attributes
+
```

Comment:
Reformat to follow similar pattern to attributes in other tags. For example, look at https://github.com/ampproject/amphtml/blob/master/builtins/amp-img.md. Attributes are formatted using h2, not list format.